config/environments/production.rb

```diff
@@ -10,9 +10,14 @@
   # Rake tasks automatically ignore this option for performance.
   config.eager_load = true
 
-  # Full error reports are disabled and caching is turned on.
+  # Full error reports are disabled.
   config.consider_all_requests_local       = false
+
+  # Caching is turned on and configure long cache lifetime.
   config.action_controller.perform_caching = true
+  config.public_file_server.headers = {
```

Comment:
I'm assuming this is for this item in the guidance:
>Configure and set long cache lifetime in Cache-Control headers on CSS and JS assets. This will improve speed on repeat visits to the application. Cache busting hashes are already in place on CSS and JS files, avoiding users receiving stale assets. 

Was 6 months just a good default you found?

Reply:
honestly I forget where I came up with this number - pretty sure I talked to @holmesie about it? this was a few weeks ago at this point so I don't really remember